### PR TITLE
feat(global): Move RXJS-utils to repository

### PR DESCRIPTION
**Pull request by IbenTesara**

### CODE REVIEW

**Review by WHeirstrate — APPROVED** [review]

File: libs/rxjs/utils/vite.config.ts
Comment (DenisValcke) [inline libs/rxjs/utils/vite.config.ts:1]: I see that the vite static copy plugin has been removed from the config. This is needed to include the `docs` folder in the bundle to make sure the links to the per-feature README files are available in npm/github.

File: project.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"name": "workspace",
+	"$schema": "node_modules/nx/schemas/project-schema.json",
+	"targets": {
+		"local-registry": {
+			"executor": "@nx/js:verdaccio",
```
Comment (DenisValcke) [inline project.json:6]: So the local registry uses verdaccio? That's pretty clever tbh.

File: libs/rxjs/utils/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+			transformMixedEsModules: true,
+		},
+		lib: {
+			// Could also be a dictionary or array of multiple entry points.
+			entry: 'src/index.ts',
+			name: 'rxjs-utils',
+			fileName: 'index',
+			// Change this to the formats you want to support.
+			// Don't forget to update your package.json as well.
+			formats: ['es', 'cjs'],
```
Comment (DenisValcke) [inline libs/rxjs/utils/vite.config.ts:37]: I think `umd` is missing here.

**Review by DenisValcke — APPROVED** [review]

File: project.json (hunk 1 above)
Comment (IbenTesara) [reply project.json:6]: All Nx here, so no idea what the inner workings are but glad you're happy :D

**Pull request merged**